kautomator/src/main/kotlin/com/kaspersky/components/kautomator/dsl/check/UiCheckableAssertions.kt

```diff
@@ -23,24 +24,8 @@ interface UiCheckableAssertions : UiBaseAssertions {
         view.check(CheckableAssertionType.IS_NOT_CHECKED) { assertThat(isChecked).isFalse() }
     }
 
-    /**
-     * Checks if the view is checkable
-     */
-    fun isCheckable() {
```

Comment:
Removed this method due to their uselessness. Checkable actions should be applied to views that implement Checkable interface.